etc/org.opencastproject.security.urlsigning.provider.impl.GenericUrlSigningProvider.cfg

```diff
@@ -25,5 +27,6 @@
 # key.demoKeyOne.organization=mh_default_org
 
 # key.demoKeyTwo.secret=6EDB5EDDCF994B7432C371D7C274F
-# key.demoKeyTwo.url=http://download.opencast.org/custom
+# key.demoKeyTwo.url.http=http://download.opencast.org/custom
+# key.demoKeyTwo.url.https=https://download.opencast.org/custom
 # key.demoKeyTwo.organization=mh_default_org
```

Comment:
Should there be an example in here, demonstrating the multiple URLs, like in the docs? I think the examples here can just straight up be copied from the docs.

Reply:
Adjusted example to example used in documentation

Reply:
Using same example as in docs now.